### PR TITLE
testing custom gapps package feature

### PR DESCRIPTION
**Pull request by mithunpk97**

### CODE REVIEW

File: 11/mpk.config [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+DigitalWellbeing=0
+VancedManager=0
+GoogleMessages=1
+GoogleDialer=0
+GoogleContacts=0
+CarrierServices=0
+GoogleClock=0
+
+# Set SetupWizard=0 if you want to skip installing all packages belonging to SetupWizard Package
+SetupWizard=0
```
Comment (nikhilmenghani) [inline 11/mpk.config:76]: If you will set the AppSet=0 their packages will also be ignored!

If you want SetupWizard in your package, make sure you set SetupWizard=1 and all the packages belonging to it =1

Comment (nikhilmenghani) [reply 11/mpk.config:76]: Also, while sending the pull request, make sure you don't delete placeholder file. Please make these changes and resend the pull request.

Comment (mithunpk97) [reply 11/mpk.config:76]: thanks for the reply. I dont want the setupwizard. also place holder in 11 folder already deleted. only want 11 build so do I need to delete place holder in 10 folder ?

**Comment by nikhilmenghani** [comment]
Also, while sending the pull request, make sure you don't delete placeholder file. Please make these changes and resend the pull request.

**Comment by nikhilmenghani** [comment]
Closing this pull request as mpk.config is already merged via a different pull request

**Pull request closed without merging**